styles.css

```diff
@@ -80,9 +291,157 @@ body {
   color: #6070ff;
 }
 
-.logo {
+.get-resume > button {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
+  border-color: #6070ff;
+  margin-top: 20px;
+  background-color: white;
+  border-radius: 8px;
+  display: flex;
+}
```

Comment:
To follow the Figma design, please change your 'Get My Resume' button styling and add the button effects (hover, active, etc).

Figma button:
![image](https://user-images.githubusercontent.com/42799579/193400239-997ac412-d2a3-4fca-b7c7-6258c4130c76.png)

Your button:
![image](https://user-images.githubusercontent.com/42799579/193400251-7d6239b3-50bf-4c41-8707-e6d8cd62cf30.png)

Figma button effects:
![image](https://user-images.githubusercontent.com/42799579/193400304-cc27e58b-76cf-4320-83a6-1476c31fb245.png)

Please take into account that the objective of this project is to guide you on following design requirements. Imagine Figma as a client, who is asking you for an exact template of a website. 😉